cmd/zstdseek/main.go

```diff
@@ -60,13 +62,27 @@ func main() {
 		logger.Fatal("verify can't be used with stdout output")
 	}
 
+	bar := progressbar.DefaultSilent(0, "")
+
 	var input io.ReadCloser
 	if inputFlag == "-" {
 		input = os.Stdin
 	} else {
-		if input, err = os.Open(inputFlag); err != nil {
+		file, err := os.Open(inputFlag)
+		if err != nil {
 			logger.Fatal("failed to open input", zap.Error(err))
 		}
+
+		if progressFlag {
+			stat, _ := file.Stat()
+
+			bar = progressbar.DefaultBytes(
+				stat.Size(),
```

Comment:
you'll need to pass -1 to convert progressbar to spinner.  But if `file.Stat()` fails, it will return 0 as `.Size()`.